src/cli.rs

```diff
@@ -0,0 +1,19 @@
+use clap::Parser;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[clap(version, about, long_about = None)]
+pub struct CliArgs {
+    #[clap(short, long)]
+    pub config_path: Option<PathBuf>,
+}
+
+pub enum CliCommand {
+    ConfigPath(PathBuf),
+}
+///This function checks for any commands passed to the program.
+///Return the arguments passed to the program if any.
```

Comment:
These docstrings are badly written and formatted:

```suggestion
/// Parses arguments passed to the program.
```